src/docs/release/breaking-changes/forgetchild-call-super

```diff
@@ -0,0 +1,84 @@
+---
+title: Element subclasses must call super in forgetChild
+description: Any element subclasses that override forgetChild will require to call super.
+---
+
+## Summary
+
+A recent global key duplication detection refactor requires `Element` subclasses which override
+`forgetChild` to call super.
+
+## Context
+
+When we encounter a global key duplication which will be cleaned up by an element rebuild later, we
```

Comment:
Can you rewrite this section to be past tense? By the time developers look at it, it's no longer the "current" implementation. It's the previous implementation.